test/functional/wallet_bumpfee.py

```diff
@@ -375,21 +346,6 @@ def test_watchonly_psbt(self, peer_node, rbf_node, dest_address):
     rbf_node.unloadwallet("watcher")
     rbf_node.unloadwallet("signer")
 
-def test_rebumping(self, rbf_node, dest_address):
```

Comment:
this seems easily updateable to feerate and worthwhile to check?

Reply:
thanks -- done